### ISSUE TITLE
[A11Y] Missing info or instruction about styling titles

### ISSUE THREAD

**Issue by Devika008** [pkp/pkp-lib#9348]
## Issue Description
// What is the nature of the issue? How does it impact users?

The form field of entering the title of the submission is now accepting styling, e.g.: bold, italic, superscript... but there is no information or instruction on how to apply it using a keyboard

## How to reproduce / Current behavior
// Step-by-step instructions explaining how to reproduce the problem. The environment required, explicit identification of the component in question

1. Create a submission
2. Go to the "Make submission: Details" step (1)
3. Move the focus to icon element beside "Title" field
4. Using keyboard interact with the field 

## What application are you using?
 OJS, OMP or OPS version X.X.X
 
 OJS 3.4

## Stack used 

- Device: Laptop
- OS: MacOS
- Browser: Chrome
- Version: 117

## Testing method
- Manual (keyboard)

## Action
// Describe action that can be taken to remediate the issue. 

Add instructions about the feature and how to style using keyboard only as well. When we do this, we will meet the WCAG Criteria of [3.3.2 Labels or Instructions - A](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html)

## Additional information
// Please add any screenshots, code snippets, or other information we can use to investigate this issue.

![image](https://github.com/pkp/pkp-lib/assets/112014397/deff6201-6567-4bdd-8faf-b46db643f607)
